Update css.md - amend sample of incorrect values

secan opened the pull request:
![wp_css-md_screenshot](https://user-images.githubusercontent.com/13051113/88164131-fed46a00-cc13-11ea-8d99-7fa000058639.png)

In line 9-13 of the sample it looks like the text formatting is incorrect and the information about not using unit for line height is missing

ntwb commented:
Thanks, I'll have to take a closer look at why the `</em>` is showing there, I suspect the tags are not balanced elsewhere in the file, I'll try to get to that this weekend

secan commented:
> Thanks, I'll have to take a closer look at why the `</em>` is showing there, I suspect the tags are not balanced elsewhere in the file, I'll try to get to that this weekend

I think it might have something to do with the `*` sign used in the comments (`/*` and `*/`); it appears that the closing asterisk in the first comment is somehow converted into an opening `<em>` tag and the opening one in the following comment becomes the closing `</em>` tag. It is almost like the asterisk is randomly treated as some sort of markdown.

I noticed the same issue appears in the same page, in the section "commenting" (lines 271-276 of _css.md_):
![wp_css-coding-standards_commenting](https://user-images.githubusercontent.com/13051113/88291943-45e85b00-ccf9-11ea-9c0b-47ec83a6d290.png)

Finally, a similar issue is present in the section "media queries" (lines 241-246 of _css.md_), where GitHub opinionatedly add some `<pre>` and `<code>` tags:
![wp_css-coding-standards_media-queries](https://user-images.githubusercontent.com/13051113/88292454-f0607e00-ccf9-11ea-81e4-7fa0807362bb.png)

secan commented:
I added committed a new version of "css.md" where I escaped all asterisks in the code snippets:
`/*` becomes `/\*`
`*/` becomes`\*/`
this should fix the issue with the `<em>` tags.

I also replaced the pre-formatted text in the "Media Queries" code sample with non formatted text, which should solve the issue of the `<pre>` and `<code>` tags appearing in the sample

ntwb commented:
Thanks @secan, sorry for the delay, and yes you are most likely correct, as the `*` is used for `<em>` in markdown, which the pages are rendered as markdown and parsed through the Jetpack Markdown Parser this is highly likely the cause, will merge this and see how it looks 💯

ntwb commented:
This resulted in:

<img width="584" alt="image" src="https://user-images.githubusercontent.com/1016458/89600091-8a045f80-d8a4-11ea-99e6-7698c6c22a99.png">

I created #41 to use HTML Entities and will see what happens 🤞🏼
